collaboration/01-sig-and-wg.md

```diff
@@ -66,7 +67,6 @@ To standardize the Special Interest Group and Working Group efforts, create maxi
 ## General guidelines
 
 All Kyma SIG and WG members agree to these rules:
-* Follow the Kyma [Manifesto](https://kyma-project.github.io/community/).
 * Follow the official project guidelines defined in the [**Contributing rules**](/contributing/#contributing-rules-contributing-rules) repository.
```

Comment:
Hm maybe we can link here: https://kyma-project.io/community/contributing/ ? I would also remove `repository`.

Reply:
I will remove `repository` but the link seems to be fine.